packages/server/modules/workspaces/tests/integration/workspaceSeat.spec.ts

```diff
@@ -112,6 +116,9 @@ describe('Workspace workspaceSeat services', () => {
     const sut = ensureValidWorkspaceRoleSeatFactory({
       createWorkspaceSeat,
       getWorkspaceUserSeat,
+      getWorkspaceDefaultSeatType: getWorkspaceDefaultSeatTypeFactory({
```

Comment:
50% of the PR (and PRs generally) are the same factory DI setup adjustments

(this is just me venting again about how much I hate this manual DI setup)